app/src/main/kotlin/nl/ndat/tvlauncher/ui/component/card/AppCard.kt

```diff
@@ -53,14 +71,31 @@ fun AppCard(
 			.width(160.dp)
 			.focusable(true, interactionSource)
 			.indication(interactionSource, FocusScaleIndication(1.125f))
-			.clickable(enabled = launchIntentUri != null) {
-				if (launchIntentUri != null) context.startActivity(
-					Intent.parseUri(
-						launchIntentUri,
-						0
+			.onKeyEvent(onKeyEvent = {
+				if (it.nativeKeyEvent.action == KeyEvent.ACTION_UP)
+					when(it.nativeKeyEvent.keyCode) {
+						KeyEvent.KEYCODE_MENU -> {
+							expanded.value = true;
+							true
+						}
+						else -> false
+					}
+				else false;
+			})
```

Comment:
I feel like we should add our own Modifier extension function to wrap this key event & the long click so we can do something like `.onMenu { expanded = true }`